types/email-templates/tsconfig.json

```diff
@@ -2,7 +2,8 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
```

Comment:
Adding this `lib` entry along with adding `juice` dependency breaks my build: https://github.com/PureBhaktiArchive/audioseva/runs/2235868573#step:8:8. My project is a purely backend project, and I don't need and don't want to add `DOM` in `tsconfig`.

Is there any way of making the project build with `@types/email-templates v8` in such cases?

Reply:
OH, it points to a bug in `juice` IMO, it should depend on Cheerio supported `Element`.
Here is a PR that removes that direct dependency, you should not see this error again:
#52180 
Also can you try skip lib check option in your tsconfig? Would it help?

Reply:
Thanks for the attention to the problem we faced.

With v8.0.3 (with #52180 included) the error stays the same. Thus, I'm blaming another change here to be the cause — namely adding explicit dependency on `juice` package.

At the same time setting `skipLibCheck` to `true` in my project eliminates the issue. Sounds like a temporary workaround.